core/src/main/java/org/elasticsearch/Version.java

```diff
@@ -138,11 +138,15 @@
     public static final Version V_5_5_1 = new Version(V_5_5_1_ID, org.apache.lucene.util.Version.LUCENE_6_6_0);
     public static final int V_5_5_2_ID = 5050299;
     public static final Version V_5_5_2 = new Version(V_5_5_2_ID, org.apache.lucene.util.Version.LUCENE_6_6_0);
-    public static final int V_5_5_3_ID_UNRELEASED = 5050399;
-    public static final Version V_5_5_3_UNRELEASED = new Version(V_5_5_3_ID_UNRELEASED, org.apache.lucene.util.Version.LUCENE_6_6_0);
-    public static final int V_5_6_0_ID_UNRELEASED = 5060099;
-    public static final Version V_5_6_0_UNRELEASED = new Version(V_5_6_0_ID_UNRELEASED, org.apache.lucene.util.Version.LUCENE_6_6_0);
-    public static final Version CURRENT = V_5_6_0_UNRELEASED;
+    public static final int V_5_5_3_ID = 5050399;
+    public static final Version V_5_5_3 = new Version(V_5_5_3_ID, org.apache.lucene.util.Version.LUCENE_6_6_0);
+    public static final int V_5_5_4_ID_UNRELEASED = 5050499;
```

Comment:
Since we've released 5.6.0 I think we can skip a constant for 5.5.4.

Reply:
I assume it also means I do not have to add the `core/src/test/resources/indices/bwc/*-5.5.3.zip` too ?

Reply:
I believe you should keep the bwc zip around.